hangupsbot/plugins/restrictedadd.py

```diff
@@ -49,27 +53,42 @@ def _botkeeper_list(bot, conv_id):
 @asyncio.coroutine
 def _check_if_admin_added_me(bot, event, command):
     bot_id = bot._user_list._self_user.id_
+    
     if event.conv_event.type_ == hangups.MembershipChangeType.JOIN:
         if bot_id in event.conv_event.participant_ids:
             # bot was part of the event
             initiator_user_id = event.user_id.chat_id
 
-            # check if botkeeper added, including self (ie, add by link)
-            if initiator_user_id in _botkeeper_list(bot, event.conv_id):
-                logger.info("botkeeper added me to {}".format(event.conv_id))
+            #if initiator_user_id != bot_id (i.e the bot account didn't join via link)
+            if initiator_user_id != bot.user_self()["chat_id"]:
+
+                # check if botkeeper added, including self (ie, add by link)
```

Comment:
Not sure I can - Need to check:
- Someone joined something
- Bot was involved
- Didn't add itself
- Did botkeeper add?
- If no, do you want to be notified about it?